lib/code_corps/stripe_service/stripe_connect_subscription.ex

```diff
@@ -112,7 +106,10 @@ defmodule CodeCorps.StripeService.StripeConnectSubscriptionService do
     |> Repo.insert
   end
 
-  defp to_create_attributes(%StripeConnectCard{} = card, %StripeConnectCustomer{} = customer, %StripeConnectPlan{} = plan, quantity) do
+  defp to_create_attributes(
+    %StripeConnectCard{} = card, %StripeConnectCustomer{} = customer,
+    %StripeConnectPlan{} = plan, %{"quantity" => quantity}
```

Comment:
Do we need to pattern match here for `StripeConnectCard` and `StripeConnectPlan` etc? This is the only method we have by this name, so the pattern matching seems unnecessary when we could simply do:
`defp to_create_attributes(card, customer, plan, %{"quantity" => quantity)`